test/integration/contract.ts

```diff
@@ -435,7 +525,7 @@ describe('Contract', function () {
       )
       queryExtend.result.returnType.should.be.equal('ok')
       const oracleExtended = await aeSdk.getOracleObject(oracleId)
-      oracleExtended.ttl.should.be.equal(oracle.ttl + 50)
+      oracleExtended.ttl.should.be.equal(parseInt(oracle.ttl) + 50)
```

Comment:
```suggestion
      oracleExtended.ttl.should.be.equal(+oracle.ttl + 50)
```
unary plus is more fragile than `parseInt`, it suits better for tests